cpp/perspective/CMakeLists.txt

```diff
@@ -125,6 +125,7 @@ option(CMAKE_BUILD_TYPE "Release/Debug build" RELEASE)
 option(PSP_WASM_BUILD "Build the WebAssembly Project" ON)
 option(PSP_CPP_BUILD "Build the C++ Project" OFF)
 option(PSP_PYTHON_BUILD "Build the Python Bindings" OFF)
+option(PSP_PYTHON_BUILD_OMIT_BINDING "Build the C++ project, omit the python. FOR CI ONLY" OFF)
```

Comment:
What are these `cmake` changes for?  I don't see them used

Reply:
I will probably undo these now that the build is so fast, this enables you to split the C++ build between the python version agnostic `libpsp` and the python version dependent `libbinding`